### ISSUE TITLE
Cannot translate neuron in viewer.draw - produces error

### ISSUE THREAD

**Issue by chalimou** [BlueBrain/NeuroM#417]
I used to translate synthesized neurons, so that the soma has (0,0,0) coordinates. I include an example synthesized neuron. After updating to "viewer" following the deprecating message the  function "translate" produces an error. I used the following script:

```
from neurom import ezy
from neurom.geom.transform import translate
import numpy
if __name__ == '__main__':
    n = ezy.load_neuron('../test_data/L23_PC_ID50.h5')
    m = translate(n, - numpy.asarray(n.soma.center))
    f1,a1=ezy.view(m, diameter=False)
```

Afterwards, I updated to fst and viewer like this:

```
from neurom import fst
from neurom import viewer
from neurom.geom.transform import translate
import numpy
if __name__ == '__main__':
    n = fst.load_neuron('../test_data/L23_PC_ID50.h5')
    m = translate(n, - numpy.asarray(n.soma.center))
    f1,a1=viewer.draw(m, diameter=False)
```

It produces the following error:

```
      7     n = fst.load_neuron('../test_data/L23_PC_ID50.h5')
      8     m = translate(n, - numpy.asarray(n.soma.center))
----> 9     f1,a1=viewer.draw(m, diameter=False)

/Users/chalimou/Projects/NEUROM/NeuroM/neurom/viewer.pyc in draw(obj, mode, **kwargs)
    117         tag = 'soma'
    118     else:
--> 119         raise NotDrawableError('draw not implemented for %s', obj.__class__)
    120
    121     viewer = '%s_%s' % (tag, mode)
```

[L23_PC_ID50.h5.zip](https://github.com/BlueBrain/NeuroM/files/350219/L23_PC_ID50.h5.zip)

**Comment by juanchopanza** [comment]
`neurom.geom.translate` is not implemented for new-style neurons, and unfortunately the implementation does not tell you this, letting the error propagate to the viewer. I am working on an implementation of translate at the moment. I hope to have an up-date soon.

**Comment by juanchopanza** [comment]
Fixed in #418.
